addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/GuesserData.java

```diff
@@ -19,13 +19,38 @@
  */
 package org.zaproxy.addon.paramminer;
 
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import org.parosproxy.paros.Constant;
+
 public class GuesserData {
+    private int threadCount;
+    private boolean redirectState;
+    private boolean useDefaultUrlWordList;
+    private List<String> urlDefaultList;
 
     public int getThreadCount() {
-        return 0;
+        return this.threadCount;
     }
 
     public boolean getRedirectState() {
-        return false;
+        return this.redirectState;
+    }
+
+    public void setDefaultUrlWordList() {
+        this.useDefaultUrlWordList = true;
+    }
+
+    public boolean useDefaultUrlWordLists() {
+        return this.useDefaultUrlWordList;
+    }
+
+    public List<String> getUrlDefaultList() {
+        if (this.urlDefaultList == null) {
+            Path file = Paths.get(Constant.getZapHome(), "wordlists/small_list.txt");
```

Comment:
I'd suggest making a class constant for this, something like:
`private final string DEFAULT_WORDLIST = "wordlists/small_list.txt";`

Reply:
DEFAULT_URL WORDLIST rather :)